src/handlers/command-parser.ts

```diff
@@ -0,0 +1,51 @@
+import { Command, InvalidArgumentError } from "commander";
+import packageJson from "../../package.json";
+import { Context } from "../types";
+import { registerWallet } from "./query-wallet";
+
+export class CommandParser {
+  readonly _program;
+
+  constructor(context: Context) {
+    const program = new Command();
+    program
+      .command("/wallet")
+      .usage("<address>")
+      .argument("<address>", "Wallet address to query, e.g. 0x000000000000000000000000000000000000001", this._parseWalletAddress)
```

Comment:
```suggestion
      .argument("<address>", "Wallet address to query, e.g. 0x000000000000000000000000000000000000000", this._parseWalletAddress)
```


You can use in the earlier example as well that I commented on